### PR TITLE
update flatbuffers; Add missed watchOS as flatbuffer platform

### PR DESCRIPTION
**Pull request by cdillard-NewRelic**
This PR modifies our NRMAHandledExceptions to properly send up watchOS platform when constructing hex events in watchOS.
This PR modifies the generated code in libMobileAgent/HEx/include/Hex/generated/agent-data_generated.h to include an additional platform watchOS.

### CODE REVIEW

File: libMobileAgent/src/Hex/include/Hex/generated/agent-data_generated.h [hunk 1]
```diff
@@ -27,15 +27,17 @@ enum Platform {
   Platform_Android = 0,
   Platform_iOS = 1,
   Platform_tvOS = 2,
+  Platform_watchOS = 3,
   Platform_MIN = Platform_Android,
-  Platform_MAX = Platform_tvOS
+  Platform_MAX = Platform_watchOS
```
Comment (smalsam-newr) [inline libMobileAgent/src/Hex/include/Hex/generated/agent-data_generated.h:32]: This is a generated file. We need to change the file that generates it

**Review by smalsam-newr — CHANGES_REQUESTED** [review]

File: libMobileAgent/src/Hex/include/Hex/generated/agent-data_generated.h (hunk 1 above)
Comment (cdillard-NewRelic) [reply libMobileAgent/src/Hex/include/Hex/generated/agent-data_generated.h:32]: 🤷 I was able to report a hex with watchOS with this change.

Comment (cdillard-NewRelic) [reply libMobileAgent/src/Hex/include/Hex/generated/agent-data_generated.h:32]: The flatbuffer schema file is contained in https://source.datanerd.us/mobile/mobile_flatbuffer_schemas which has had watchOS platform added.

Comment (cdillard-NewRelic) [reply libMobileAgent/src/Hex/include/Hex/generated/agent-data_generated.h:32]: Please give me a helpful suggestion as to where the file that generates it is then. The change I made works well.

**Review by smalsam-newr — APPROVED** [review]

**Review by smalsam-newr — APPROVED** [review]

**Pull request merged**